apps/web/src/components/applicant/ApplicantProfile.tsx

```diff
@@ -94,7 +94,7 @@ export const ApplicantProfile = () => {
             </Dialog>
           </section>
         </AclMask>
-        {!applicant?.deleted_date && (
+        {!applicant?.deleted_date ? (
```

Comment:
This change makes us unable to reactivate applicants. Is it ok for all users?  If it's just for HA users, we might need a condition to check the logged-in user.

Reply:
The intent is for the applicant to be archived permanently. They are still accessible through the "show hidden applicants" toggle, and their data is still visible in the extracts.